internal/providers/terraform/aws/rds_cluster.go

```diff
@@ -0,0 +1,213 @@
+		snapShotStorageSizeGB := decimal.Zero
+		if u != nil && u.Get("backup_snapshot_size_gb").Exists() {
+			snapShotStorageSizeGB = decimal.NewFromFloat(u.Get("backup_snapshot_size_gb").Float())
+		}
+
+		totalBackupStorageGB := calculateBackupStorage(snapShotStorageSizeGB, backupStorageRetention)
+
+		if totalBackupStorageGB.GreaterThan(decimal.Zero) {
+			costComponents = append(costComponents, auroraBackupStorageCostComponent(region, totalBackupStorageGB, databaseEngineStorageType))
+		}
+	}
+
+	backtrackChangeRecords := decimal.Zero
+	if u != nil && u.Get("backtrack_change_records").Exists() {
+		backtrackChangeRecords = decimal.NewFromInt(u.Get("backtrack_change_records").Int())
+	}
+
+	backtrackPeriod := decimal.Zero
+	if u != nil && u.Get("backtrack_change_record_hrs").Exists() {
+		backtrackPeriod = decimal.NewFromInt(u.Get("backtrack_change_record_hrs").Int())
+	}
+
+	totalBacktrackChangeRecords := backtrackChangeRecords.Mul(backtrackPeriod)
+	if totalBacktrackChangeRecords.GreaterThan(decimal.Zero) {
+		costComponents = append(costComponents, &schema.CostComponent{
+			Name:            "Backtrack",
+			Unit:            "Record-hours",
+			UnitMultiplier:  1000000,
+			MonthlyQuantity: decimalPtr(totalBacktrackChangeRecords),
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Service:       strPtr("AmazonRDS"),
+				ProductFamily: strPtr("System Operation"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", Value: strPtr("Aurora:BacktrackUsage")},
+				},
+			},
+		})
+	}
+
+	snapshotExportSizeGB := decimal.Zero
+	if u != nil && u.Get("snapshot_export_size_gb").Exists() {
+		snapshotExportSizeGB = decimal.NewFromFloat(u.Get("snapshot_export_size_gb").Float())
+	}
+
+	if snapshotExportSizeGB.GreaterThan(decimal.Zero) {
+		costComponents = append(costComponents, &schema.CostComponent{
+			Name:            "Snapshot export",
+			Unit:            "GB",
+			UnitMultiplier:  1,
+			MonthlyQuantity: decimalPtr(snapshotExportSizeGB),
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Service:       strPtr("AmazonRDS"),
+				ProductFamily: strPtr("System Operation"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "databaseEngine", Value: databaseEngine},
+					{Key: "usagetype", Value: strPtr("Aurora:SnapshotExportToS3")},
+				},
+			},
+		})
+	}
+
+	return &schema.Resource{
+		Name:           d.Address,
+		CostComponents: costComponents,
+	}
+}
+
+func auroraStorageCostComponent(region string, u *schema.UsageData, databaseEngineStorageType *string) []*schema.CostComponent {
+	storageGB := decimal.Zero
+	if u != nil && u.Get("storage_gb").Exists() {
+		storageGB = decimal.NewFromFloat(u.Get("storage_gb").Float())
+	}
+
+	writeRequestsPerSecond := decimal.Zero
+	if u != nil && u.Get("read_request_secs").Exists() {
+		writeRequestsPerSecond = decimal.NewFromInt(u.Get("write_request_secs").Int())
```

Comment:
Should this be `write_requests_per_sec` since this isn't multiplied by seconds? Same for `read_requests_per_sec`